<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtEEG-MAP</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon.png</normaloff>
    </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>553</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Domain</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioTime">
            <property name="text">
             <string>Tegangan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFrequencyMultiple">
            <property name="text">
             <string>Frekuensi Ganda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFrequencySingle">
            <property name="text">
             <string>Frekuensi Tunggal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Elektroda</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkboxFP1">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>FP1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxFP2">
            <property name="text">
             <string>FP2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxF3">
            <property name="text">
             <string>F3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxF4">
            <property name="text">
             <string>F4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxC3">
            <property name="text">
             <string>C3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxC4">
            <property name="text">
             <string>C4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxP3">
            <property name="text">
             <string>P3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxP4">
            <property name="text">
             <string>P4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxO1">
            <property name="text">
             <string>O1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxO2">
            <property name="text">
             <string>O2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxF7">
            <property name="text">
             <string>F7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxF8">
            <property name="text">
             <string>F8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxT3">
            <property name="text">
             <string>T3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxT4">
            <property name="text">
             <string>T4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxT5">
            <property name="text">
             <string>T5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxT6">
            <property name="text">
             <string>T6</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Pita Frekuensi</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkboxGamma">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxBeta">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxAlpha">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxTheta">
            <property name="text">
             <string>Theta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxDelta">
            <property name="text">
             <string>Delta</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Dokumen</string>
    </property>
    <widget class="QMenu" name="menuConnect_do_device">         
     <property name="title">
    <string>&amp;Perangkat</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuSpikes">
    <property name="title">
     <string>Plot</string>
    </property>
    
    <addaction name="actionSpike_sorting"/>
    <addaction name="actionSpike_detecting"/>
    <addaction name="actionTopoplot"/>
    <addaction name="actionSpectrum"/>
    <addaction name="actionClustering"/>                  
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Ekstraksi Fitur</string>
    </property>
    <addaction name="actionFeature_extraction"/>
    <addaction name="actionPCA_extraction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSpikes"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSpike_sorting"/>
   <addaction name="actionSpike_detecting"/>
   <addaction name="actionFeature_extraction"/>
   <addaction name="actionPCA_extraction"/>
   <addaction name="actionTopoplot"/>
   <addaction name="actionSpectrum"/>
   <addaction name="actionClustering"/>                 
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
    <normalon>:/icon2/open2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Buka Dokumen</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Keluar</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Bantuan</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pengaturan</string>
   </property>
  </action>
  <action name="actionSpike_sorting">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pengurutan Spike</string>
   </property>
  </action>
  <action name="actionSpike_detecting">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Deteksi Spike</string>
   </property>
  </action>
  <action name="actionFeature_extraction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>FastICA</string>
   </property>
  </action>
  <action name="actionPCA_extraction">
   <property name="enabled">
    <bool>false</bool>  
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>  
    </iconset>
   </property>
   <property name="text">
    <string>PCA</string>  
   </property>
  </action>
  <action name="actionTopoplot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Scalp Map</string>
   </property>
   </action>
  <action name="actionSpectrum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Spektrum</string>
   </property>
</action>
  <action name="actionClustering">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Klastering</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph/widgets/GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>